kroxylicious-docs/docs/record-encryption-quickstart/index.adoc

```diff
@@ -1,8 +1,11 @@
 :experimental:
-:kafka-version: 4.0.0
 :vault-chart-version: 0.30.1
 :localstack-chart-version: 0.6.24
+// A Renovate rule will update the strimzi-version but not kafka-version. Update the kafka-version to the point
+// at the latest kafka version supported by that strimzi.
 :strimzi-version: 0.47.0
+:kafka-version: 4.0.0
+:kafka-image: quay.io/strimzi/kafka:{strimzi-version}-kafka-{kafka-version}
```

Comment:
I couldn't find a reasonable way to get renovate to update both strimzi version and kafka version in one go.
I've left the upgrading of a kafka part as a manual step.